tests/testAutopas/tests/autopasInterface/AutoPasInterfaceTest.cpp

```diff
@@ -20,9 +20,8 @@
 #include "testingHelpers/commonTypedefs.h"
 
 extern template class autopas::AutoPas<Molecule>;
-using LJFunctorGlobals =
-    mdLib::LJFunctor<Molecule, /* shifting */ true, /*mixing*/ false, autopas::FunctorN3Modes::Both,
-                     /*globals*/ true>;
+using LJFunctorGlobals = LJFunctorType</* shifting */ true, /*mixing*/ false, autopas::FunctorN3Modes::Both,
+                                       /*globals*/ true>;
```

Comment:
maybe move this alias to `commonTypedefs`

Reply:
See `6aaea17f `